recipes/ocl/build.sh

```diff
@@ -0,0 +1,19 @@
+mkdir build -p
+cd build 
+
+if [[ ${HOST} =~ .*linux.* ]]; then
```

Comment:
Use `target_platform` variable here

Reply:
thanks, done.

Reply:
btw.: I didn't find this in the docs. Should it be added?

Reply:
HOST is going to be renamed soon. I can't find the discussion at the moment

Reply:
hmm weird.  Yeah, I missed that too. Docs would be great for that

Reply:
Copying @isuruf's comment here for continuity.

> Found the discussion [conda/conda#7031](https://github.com/conda/conda/issues/7031)